.github/workflows/deploy.yml

```diff
@@ -8,7 +8,6 @@ on:  # yamllint disable-line rule:truthy
 
 jobs:
     deploy:
-        needs: [linter, test]
```

Comment:
I think linter and tests should pass before we start deployment otherwise we deploy some crap. Am I wrong ?

Reply:
Actually it is OK. We run tests in PR and we don't allow merging without beeing updated with master